lib/src/rewrite.rs

```diff
@@ -569,6 +569,13 @@ impl<'settings, 'repo> DescendantRebaser<'settings, 'repo> {
             // The commit is already in place.
             return Ok(());
         }
+        assert_eq!(
+            (self
+                .rebased.get(&old_commit_id), self
+                .parent_mapping.get(&old_commit_id)),
+            (None, None),
```

Comment:
nit: `.contains_key()`

Reply:
I think the reason the assertion looked this way was so we get to see what value is in the map if it fails.